deploy/default-rbac.yaml

```diff
@@ -38,12 +38,20 @@ rules:
 # Allow actions on basic Kubernetes objects
 - apiGroups: [""]
   resources:
-  - configmaps
-  - secrets
-  - pods
   - services
   - serviceaccounts
   - serviceaccounts/token
-  - endpoints
+  verbs: ["get"]
+- apiGroups: [""]
+  resources:
   - events
-  verbs: ["*"]
+  verbs: ["create"]
+- apiGroups: [""]
+  resources:
+  - endpoints
+  - secrets
+  verbs: ["list", "get", "update"]
```

Comment:
Thanks for the PR, agree on narrowing down the rbac policy examples. I think permissions to create secrets might be missing here

Reply:
I'll update that and also the helm charts version tomorrow.